server/config/config.go

```diff
@@ -164,6 +164,10 @@ type Config struct {
 	ReplicationMode ReplicationModeConfig `toml:"replication-mode" json:"replication-mode"`
 
 	EnableAuditMiddleware bool
+
+	MaxRegionSize uint64 `toml:"max-region-size" json:"max-region-size"`
```

Comment:
Does this PR need to be closed? We don't need the configuration anymore?

Reply:
yes